### PR TITLE
DOCS-2360 Filtration mode article update

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 1]
```diff
@@ -91,23 +91,11 @@ In this example, the filtration modes are defined for the resources as follows:
...
-* [**Blocking**](#available-filtration-modes)
+You can define the general filtration mode for all incoming requests in **Settings** → **General** in the [US](https://us1.my.wallarm.com/settings/general) or [EU](https://my.wallarm.com/settings/general) Cloud. This setting has the same scope as the `wallarm_mode` directive value defined in the `http` block in the configuration file.
     
 ![The general settings tab](../images/configuration-guides/configure-wallarm-mode/en/general-settings-page-with-safe-blocking.png)
 
-!!! info "The Wallarm Cloud and filtering node synchronization"
-    The rules defined in Wallarm Console are applied during the Wallarm Cloud and filtering node synchronization process, which is conducted once every 2‑4 minutes.
-
-    [More details on the filtering node and Wallarm Cloud synchronization configuration →](configure-cloud-node-synchronization-en.md)
+The general filtration mode setting is represented as **Set filtration mode** [default](../user-guides/rules/rules.md#default-rules) rule on the **Rules** tab. Note that local rules created on this tab have higher priority.
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:98]: why local? you already defined terminology for rules of this level in the rules overview

Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:98]: Rules is not a tab, it is a section

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 2]
```diff
@@ -117,43 +105,22 @@ You can fine-tune the filtration mode for processing requests that meet your cus
...
+To do so, set the **Set filtration mode** rule as displayed on the screenshot:
 
 ![Setting traffic filtration mode][img-mode-rule]
 
-**API calls to create the rule**
-
-To create the filtration mode rule, you can [call the Wallarm API directly](../api/overview.md) besides using the Wallarm Console UI. Below is the example of the corresponding API call.
-
-The following request will create the rule setting the node to filter traffic going to the [application](../user-guides/settings/applications.md) with ID `3` in the monitoring mode.
+**API call to create the rule**
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:121]: delete this section, it is enough to mention it in the api ref

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 3]
```diff
@@ -117,43 +105,22 @@ You can fine-tune the filtration mode for processing requests that meet your cus
...
 
-**If** the following conditions take place:
+    [More details on the filtering node and Wallarm Cloud synchronization configuration →](configure-cloud-node-synchronization-en.md)
 
-* new user registration is available at *example.com/signup*
-* it is better to overlook an attack than to lose a customer
+**Example: disabling request blocking during user registration**
 
-**Then**, to create a rule disabling blocking during user registration
+Let us say a new user registration for your application is available at `example.com/signup`. As it is better to overlook an attack than to lose a customer, whatever blocking measures are applied for your application, it is better to disable blocking during user registration.
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:116]: You can create and apply the rule both in the Attacks and Rules section of Wallarm Console. -> You can create and apply the rule both in the Attacks and Rules sectionS of Wallarm Console.

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 4]
```diff
@@ -117,43 +105,22 @@ You can fine-tune the filtration mode for processing requests that meet your cus
 
 --8<-- "../include/waf/features/rules/rule-creation-options.md"
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:106]: in this text, you say that rules can be created both from the Attacks and Rules section. however, above you say only about the rules section

I think, there is no need to have the "**Creating and applying the rule**" section section as a separate one. just write a good intro about rule creation and give an example of a rule

Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-wallarm-mode.md:106]: also, there is currently no explanation on how to configure parts of a rule

File: docs/latest/admin-en/configure-wallarm-mode.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/latest/admin-en/configure-wallarm-mode.md:98]: We do have the "default" rule term. But for those that are not default, I cannot see any particular term anywhere.

File: docs/latest/admin-en/configure-wallarm-mode.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/admin-en/configure-wallarm-mode.md:106]: All fixed. If the current description of how the rule is created is valid, I think we'll need to change other rule articles and the main Rules article correspondingly.

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 5]
```diff
@@ -89,26 +89,31 @@ In this example, the filtration modes are defined for the resources as follows:
       2. The `block` mode is applied to the requests sent to the virtual server `SERVER_C` that contain the `/main/login` path.
       3. The `safe_blocking` mode is applied to the requests sent to the virtual server `SERVER_C` that contain the `/main/reset-password` path.
 
-### Setting up the general filtration rule in Wallarm Console
+### Setting up general filtration rule in Wallarm Console
 
 You can define the general filtration mode for all incoming requests in **Settings** → **General** in the [US](https://us1.my.wallarm.com/settings/general) or [EU](https://my.wallarm.com/settings/general) Cloud. This setting has the same scope as the `wallarm_mode` directive value defined in the `http` block in the configuration file.
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:94]: let's delete the phrase "This setting has the same scope as the `wallarm_mode` directive value defined in the `http` block in the configuration file." - what does it mean? wallarm_mode and general rule in the UI are different settings

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 6]
```diff
@@ -89,73 +89,41 @@ In this example, the filtration modes are defined for the resources as follows:
...
+### Setting up filtration rules in Rules section
 
-**Default instance of rule**
+You can fine-tune the filtration mode for processing requests that meet your custom conditions in the **Rules** section of Wallarm Console. These rules have higher priority than the [general filtration rule set in Wallarm Console](#setting-up-general-filtration-rule-in-wallarm-console).
 
-Wallarm automatically creates the instance of the `Set filtration mode` rule on the [default](../user-guides/rules/rules.md#default-rules) level. The system sets its value on the basis of [general filtration mode](#setting-up-the-general-filtration-rule-in-wallarm-console) setting.
+To create a new filtration mode rule:
 
-This instance of the rule cannot be deleted. To change its value, modify [general filtration mode](#setting-up-the-general-filtration-rule-in-wallarm-console) setting of the system.
+1. [Call the Wallarm API directly](../api/request-examples.md#create-the-rule-setting-filtration-mode-to-monitoring-for-the-specific-application) or:
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:106]: this should not be the first item as this is not the prefered option

Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-wallarm-mode.md:106]: just put it as an alternative option after the list (1 sentence)

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 7]
```diff
@@ -27,9 +27,9 @@ The filtration mode can be configured in the following ways:
     !!! warning "Support of the `wallarm_mode` directive on the CDN node"
         Please note that the `wallarm_mode` directive cannot be configured on the [Wallarm CDN nodes](../installation/cdn-node.md). To configure the filtration mode of the CDN nodes, please use other available methods.
 * Define the general filtration rule in Wallarm Console
-* Create a filtration mode rule in the **Rules** section of Wallarm Console
+* Create a filtration mode rule in the **Rules** or **Attacks** sections of Wallarm Console
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:30]: API Discovery section?

File: docs/latest/admin-en/configure-wallarm-mode.md [hunk 8]
```diff
@@ -91,30 +91,31 @@ In this example, the filtration modes are defined for the resources as follows:
...
 
-You can define the general filtration mode for all incoming requests in **Settings** → **General** in the [US](https://us1.my.wallarm.com/settings/general) or [EU](https://my.wallarm.com/settings/general) Cloud. This setting has the same scope as the `wallarm_mode` directive value defined in the `http` block in the configuration file.
+You can define the general filtration mode for all incoming requests in **Settings** → **General** in the [US](https://us1.my.wallarm.com/settings/general) or [EU](https://my.wallarm.com/settings/general) Cloud.
     
 ![The general settings tab](../images/configuration-guides/configure-wallarm-mode/en/general-settings-page-with-safe-blocking.png)
 
 The general filtration mode setting is represented as **Set filtration mode** [default](../user-guides/rules/rules.md#default-rules) rule in the **Rules** section. Note that local rules created in this section have higher priority.
 
-### Setting up filtration rules in Rules section
+### Setting up specific filtration rules in Wallarm Console
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-wallarm-mode.md:100]: not specific. this term is too geenral. they are for specific endpoints/applications. Setting up endpoint-targeted filtration rule in Wallarm Console maybe?
btw - recheck that there are no links from other documents to this section with the old name

File: docs/latest/admin-en/configure-wallarm-mode.md (hunk 5 above)
Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-wallarm-mode.md]: not fixed

**Pull request merged**